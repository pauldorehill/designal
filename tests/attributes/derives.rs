use designal::Designal;

#[derive(Designal)]
pub struct HumanBeanMutable {
    #[designal(derive = "Debug")]
    taste: u8,
}

fn main() {}
