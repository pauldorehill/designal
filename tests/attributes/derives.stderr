error: You can't derive on a field
 --> $DIR/derives.rs:5:16
  |
5 |     #[designal(derive = "Debug")]
  |                ^^^^^^
